<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtCellLifeSimulationAppClass</class>
 <widget class="QMainWindow" name="QtCellLifeSimulationAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cell Life Simulation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QtCellLifeSimulationApp.qrc">
    <normaloff>:/QtCellLifeSimulationApp/bacteria.ico</normaloff>:/QtCellLifeSimulationApp/bacteria.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCells">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionPlay_Pause"/>
    <addaction name="actionGenerate"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionGenerationSettings"/>
    <addaction name="actionLimitSettings"/>
    <addaction name="actionTimeSettings"/>
   </widget>
   <addaction name="menuCells"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay_Pause"/>
   <addaction name="actionGenerate"/>
   <addaction name="actionClear"/>
  </widget>
  <action name="actionGenerate">
   <property name="icon">
    <iconset resource="QtCellLifeSimulationApp.qrc">
     <normaloff>:/toolbar/resources/add.png</normaloff>:/toolbar/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="QtCellLifeSimulationApp.qrc">
     <normaloff>:/toolbar/resources/clear.png</normaloff>:/toolbar/resources/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionGenerationSettings">
   <property name="text">
    <string>Generation</string>
   </property>
  </action>
  <action name="actionTimeSettings">
   <property name="text">
    <string>Time</string>
   </property>
  </action>
  <action name="actionPlay_Pause">
   <property name="icon">
    <iconset resource="QtCellLifeSimulationApp.qrc">
     <normaloff>:/toolbar/resources/play.png</normaloff>:/toolbar/resources/play.png</iconset>
   </property>
   <property name="text">
    <string>Play/Pause</string>
   </property>
  </action>
  <action name="actionLimitSettings">
   <property name="text">
    <string>Limit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtCellLifeSimulationApp.qrc"/>
 </resources>
 <connections/>
</ui>
